it/src/test/java/apoc/it/core/ExportCypherEnterpriseFeaturesTest.java

```diff
@@ -129,6 +131,13 @@ public void shouldHandleTwoLabelsWithOneCompoundConstraintEach() {
     }
 
     private void assertExportStatement(String expectedStatement, String fileName) {
-        assertEquals(expectedStatement, readFileToString(new File(directory, fileName)));
+        // The constraints are exported in arbitrary order, so we cannot assert on the entire file
+        String actual = readFileToString(new File(directory, fileName));
+        System.out.println(expectedStatement);
+        System.out.println(readFileToString(new File(directory, fileName)));
+        assertTrue(actual.contains(expectedStatement));
```

Comment:
Moreover, I would change the `assertTrue` with a more explanatory code in case of error, e.g. :
```
MatcherAssert.assertThat(actual, Matchers.containsString(expectedStatement));
```